### PR TITLE
Note to Azure AD article

### PR DESCRIPTION
**Pull request by sofietoft**
Added a note to the Azure AD article based on comments in this Issue: https://github.com/umbraco/Umbraco-CMS/issues/13100

### CODE REVIEW

File: umbraco-cms/reference/security/authenticate-with-active-directory.md [hunk 1]
```diff
@@ -19,199 +19,225 @@ You need to install the `Microsoft.AspNetCore.Authentication.MicrosoftAccount` N
...
+{% endcode %}
+
+* Build and run the website. You can now login with your Azure AD credentials.
 
     ![AD Login Screen](images/AD\_Login.png)
 
+{% hint style="info" %}
+In some cases where Azure B2C does not provide an email for the user, it can be necessary to add additional code.
+
+Add the following snippet within the `backOfficeAuthenticationBuilder.SchemeForBackOffice` between lines 29 and 30 in the code sample in step one:
```
Comment (bergmania) [inline umbraco-cms/reference/security/authenticate-with-active-directory.md:93]:
```suggestion
Add the following snippet within the options of `AddMicrosoftAccount` between lines 29 and 30 in the code sample in step one:
```

File: umbraco-cms/reference/security/authenticate-with-active-directory.md [hunk 2]
```diff
@@ -19,199 +19,225 @@ You need to install the `Microsoft.AspNetCore.Authentication.MicrosoftAccount` N
...
+
+```csharp
+ // Example on how to get a different field from the user profile
+options.UserInformationEndpoint = "https://graph.microsoft.com/v1.0/me?$select=otherMails,displayName,givenName,surname,id";
+options.ClaimActions.MapCustomJson(ClaimTypes.Email, x =>
+{
+    return x.GetProperty("otherMails").EnumerateArray().First().ToString();
+});
+```
+{% endhint %}
```
Comment (bergmania) [inline umbraco-cms/reference/security/authenticate-with-active-directory.md:103]:
```suggestion
This extra code will read the user email from the microsoft graph api.

{% endhint %}
```

**Review by bergmania — COMMENTED** [review]
Small suggestions

**Comment by sofietoft** [comment]
Thanks for the review @bergmania ! 💪 

I've added the suggestions. Will merge in the changes!

**Pull request merged**